tests/unit/lms/resources/_js_config/__init___test.py

```diff
@@ -480,9 +480,11 @@ def with_no_user(self, pyramid_request):
 
 class TestAddDeepLinkingAPI:
     def test_it_adds_deep_linking_v11(self, js_config, pyramid_request):
-        pyramid_request.lti_params = {
-            "content_item_return_url": sentinel.content_item_return_url,
-        }
+        pyramid_request.lti_params.update(
```

Comment:
This assignment was replacing the whole lti_params for the test. We just want to add on top of the general fixture.